RAP/rap-statistics.qmd

```diff
@@ -105,95 +111,89 @@ While working to reach our baseline expectation of good and great practice, you
+![](../images/self-assessment.png)
 
 
 Some teams will already be looking at best practice, while others will still have work to do to achieve the department's baseline of good and great practice. We know that all teams are starting this from different points, and are here to support all teams from their respective starting positions.
 
 ---
 
-## Where we need to focus
-
-
-
-Most teams have already made progress with their production of tidy data files, and the release of the automated screener has now tied up that end point of the pipeline that we are all currently working towards. 
-
-The key now is for us to build on the work so far and focus on how we improve the quality and efficiency of our production processes up to that point. To do this, we need to make a concerted effort to standardise how we store and access our data, before then automating what we can to reduce the burden of getting the numbers ready and see the benefits of RAP. The exact meaning of this will vary within teams.
-
----
-
-## How to get started
+## How to get started with RAP
 
 Check your analysis against the Good, Great and Best practice standards under [RAP in practice](/RAP/rap-statistics.html#rap-in-practice).
 
 Measure your publication against the RAP levels using our [self assessment tool](https://rsconnect/rsc/publication-self-assessment/){target="_blank" rel="noopener noreferrer"}. This will give you a good starting point and initial points to work on to progress to the next level of RAP.
 
 Once you've assessed your publication, have a look through our guidance below to narrow down how you can get started with improving those parts of your process.
 
-The Statistics Development Team invites teams to take part in our partnership programme to develop their skills and implement RAP principles to a relevant project. Visit our page on  [getting started with the partnership programme](../learning-development/learning-development.html#Support_available) for more details.
+The [Statistics Development Team](mailto:statistics.development@education.gov.uk) invites teams to take part in our partnership programme to develop their skills and implement RAP principles to a relevant project. Partnership programmes can offer additional resource and dedicated support to your team to implement specific RAP principles. Visit our page on  [getting started with the partnership programme](../learning-development/learning-development.html#partnership-programmes) for more details.
```

Comment:
The partnership programme link doesn't work for me

Reply:
fixed!